node-zerox/src/utils.ts

```diff
@@ -104,3 +108,30 @@ export const convertPdfToImages = async ({
     console.error("Error during PDF conversion:", err);
   }
 };
+
+// Convert each page (from docx) to a png and save that image to tmp
+export const convertFileToPdf = async ({
+  localPath,
+  tempDir,
+  extension,
+}: {
+  localPath: string;
+  tempDir: string;
+  extension: string;
+}): Promise<string> => {
+  const inputBuffer = await fs.readFile(localPath);
+  const outputFilename = path.basename(localPath, extension) + ".pdf";
+  const outputPath = path.join(tempDir, outputFilename);
+
+  try {
+    const pdfBuffer = await convertAsync(inputBuffer, ".pdf", undefined);
+    await fs.writeFile(outputPath, pdfBuffer);
+    return outputPath;
+  } catch (err) {
+    console.error(
+      `Error during ${extension.toUpperCase()} to PDF conversion:`,
```

Comment:
I think extension also has  the "." in front. To simplify, let's just have it worded as "Error converting ${extension} to .pdf: "